### DISCUSSION TITLE
請協助加上嘸蝦米的輸入方式

### DISCUSSION THREAD

**Discussion by seedlcw** [osfans/android-rime#1427] [Ideas | 想法]
因為在鴻蒙系統上沒有嘸蝦米輸入法，
而拼音、雙拼、五筆等輸入法又不會，
重新學習的成本太高，
所以請求高手在trime框架下，
加上嘸蝦米輸入法，謝謝。

**Comment by spew8712** [comment]
可以參考以下rime方案
https://github.com/brianhsu/rime-liur-lua
https://github.com/hsuanyi-chou/rime-liur
至於trime前端，應該會預設調用trime.yaml裡內建的26鍵鍵盤
